src/main/java/bussiness_layer/services/AuthenticationService.java

```diff
@@ -30,7 +30,7 @@
     public Authentication authenticate(Authentication authentication) throws AuthenticationException {
         String username = authentication.getPrincipal().toString();
         String password = authentication.getCredentials().toString();
-        User user = userRepo.findOneByUsername(username);
+        User user = userRepo.findByUsername(username);
         if(user == null || !BCrypt.checkpw(password, user.getEncryptedPassword())){
             throw new BadCredentialsException("Invalid username or password");
```

Comment:
Keep this message in a final class exceptionMessages or something like that, in order to have more control over the application. Create this class in exception package. Naming convetion for static final string constants: THIS_IS_A_EXCEPTION_MESSAGE = "exception message"